lib/FrontendTool/FrontendTool.cpp

```diff
@@ -1094,6 +1094,15 @@ static void setMSFPrivateDiscriminator(IRGenOptions &IRGenOpts,
     IRGenOpts.DWARFDebugFlags += (" -private-discriminator " + PD.str()).str();
 }
 
+static Optional<bool>
+emitSIBIFNeededAfterOptimizations(CompilerInvocation &Invocation, SILModule *SM,
```

Comment:
Capital "IF", but again I think this should be merged with the other emitSIB.

Reply:
Yes, just calls serializeSIB.